.github/workflows/dev.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
           path: build-output/tel2-image.tar
   run_tests:
     strategy:
+      fail-fast: false
```

Comment:
Does this mean that we have to wait for all jobs to complete just in order to view the logs of one that failed?

Reply:
No, what this does is that if one job fails, it won't abort all the others.
But as soon as a job finished, the logs will be visible, same as before.